apis/shopper-promotions/shopper-promotions.raml

```diff
@@ -68,7 +68,7 @@ types:
     examples:
       promotion: !include Examples/promotions_result.raml
     properties:
-      count:
+      limit:
```

Comment:
it seems `count` is renamed to `limit`? is this expected? isn't this a breaking change?

Reply:
This change was made around 2 years ago where it was updated from `count` to `limit`. Technically it is a breaking change, but the API doesn't return `count` but instead returns `limit` and has been for 2 years. It might be more accurate to categorize this as a bug fix